### ISSUE TITLE
horizontal navigation

### ISSUE THREAD

**Issue by Hab5Fan31** [Hab5Fan31/using-markbot#4]
Hello,
I have been on this folder for almost 3 hours trying to figure out whats wrong and i can't seem to figure it out.
I feel as if I'm on the right track but every time i do something in my css its not showing up in the browser, it just stays the same ( plain underlined links and bullet points)
i was wondering if you could help me get back on track i really want to understand this.
I had to take screenshots of the file in order to show you because it doesn'
<img width="1680" alt="screenshot 2018-10-14 17 05 51" src="https://user-images.githubusercontent.com/43150586/46922226-eca3b180-cfd3-11e8-97cd-ecaabf2265bb.png">
<img width="1680" alt="screenshot 2018-10-14 17 05 53" src="https://user-images.githubusercontent.com/43150586/46922227-eca3b180-cfd3-11e8-9fca-eeb08f927d64.png">

t support the original file.
Thank you for your time,
Amanda Dupuis.
@thomasjbradley

**Comment by thomasjbradley** [comment]
This is the wrong repository—this Issue is connected to the `using-markbot` assignment, not the `horizontal-nav` assignment.

But I can see, from looking at the screenshots, that the CSS file isn’t connected to the HTML, on line 8 of the HTML the path to the CSS file is missing. It should be:

```html
href="css/main.css"
```

**Comment by Hab5Fan31** [comment]
Yeah I figured it out. Thank you for your time sorry about these issues, I should know these things.
Have a great evening!

On Oct 14, 2018, at 7:07 PM, Thomas J Bradley <notifications@github.com<mailto:notifications@github.com>> wrote:


This is the wrong repository—this Issue is connected to the using-markbot assignment, not the horizontal-nav assignment.

But I can see, from looking at the screenshots, that the CSS file isn’t connected to the HTML, on line 8 of the HTML the path to the CSS file is missing. It should be:

href="css/main.css"
